<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>180</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>FamilyTree</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>tree-icon-0.png</normaloff>tree-icon-0.png</iconset>
  </property>
  <property name="locale">
   <locale language="Polish" country="Poland"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>458</width>
      <height>142</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_15">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_13">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_12">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="1">
            <widget class="QPushButton" name="addFamily">
             <property name="text">
              <string>Dodaj Rodzinę</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="addPerson">
             <property name="text">
              <string>Dodaj Osobę</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QPushButton" name="printPdf">
             <property name="text">
              <string>Do PDF</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="drawTree">
             <property name="text">
              <string>Rysuj Drzewo</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="0" column="1">
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="1">
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="1" column="0">
                <widget class="QCheckBox" name="posterityCheckBox">
                 <property name="text">
                  <string>zstępni</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QCheckBox" name="ancestorsCheckBox">
                 <property name="text">
                  <string>wstępni</string>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="0" column="0">
                <widget class="QRadioButton" name="drawVertically">
                 <property name="text">
                  <string>pionowo</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QRadioButton" name="drawHorizontally">
                 <property name="text">
                  <string>poziomo</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="2">
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="1" column="0">
                <widget class="QCheckBox" name="descriptionCheckBox">
                 <property name="text">
                  <string>opis</string>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QCheckBox" name="marginCheckBox">
                 <property name="text">
                  <string>margines</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_10">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_8">
               <item row="0" column="0">
                <widget class="QFontComboBox" name="fontComboBox">
                 <property name="currentText">
                  <string>Monotype Corsiva</string>
                 </property>
                 <property name="currentFont">
                  <font>
                   <family>Monotype Corsiva</family>
                   <pointsize>12</pointsize>
                   <italic>true</italic>
                  </font>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="fontSize">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="value">
                  <number>14</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>148</width>
                 <height>13</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="peopleList">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::NoInsert</enum>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_14">
       <item row="0" column="2">
        <widget class="QPushButton" name="personsFamilies">
         <property name="text">
          <string>Rodziny</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>Zapisz</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="editPerson">
         <property name="text">
          <string>Edytuj</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="loadButton">
         <property name="text">
          <string>Wczytaj</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="deletePerson">
         <property name="text">
          <string>Usuń</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>451</width>
      <height>133</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="descriptionLayout">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="descriptionLabel">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Opis:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QSpinBox" name="descriptionFontSize">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="value">
          <number>72</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QFontComboBox" name="descriptionFont">
         <property name="currentText">
          <string>Monotype Corsiva</string>
         </property>
         <property name="currentFont">
          <font>
           <family>Monotype Corsiva</family>
           <pointsize>12</pointsize>
           <italic>true</italic>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QTextEdit" name="descriptionText">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>478</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
